### ISSUE TITLE
  Potential ACS string pool issue

### ISSUE THREAD

**Issue by coelckers** [coelckers/gzdoom#136]
http://forum.zdoom.org/viewtopic.php?f=2&t=54387

**Comment by alexey-lysiuk** [comment]
Reproduced it once, I'll to figure out how to do it reliably at least.

**Comment by alexey-lysiuk** [comment]
Yes, all steps to reproduce known to me are more or less the same.
Particular chain of events to get referenced string with zero counter is not however.
